### ISSUE TITLE
Request: Thin monoid?

### ISSUE THREAD

**Issue by leeola** [larsenwork/monoid#146]
Would it be possible to get a thin variant on Monoid? Coming from Source Code Pro Thin, Monoid Regular/Retina looks very thick. Thoughts?

For reference..

OS: OSX, Mavericks
App: iTerm2
Size: 12pt _(Comparable to Source Code Pro's 14pt)_

**Comment by larsenwork** [comment]
have you tried changing the antialiasing? (you can do that in atom and other similar editors)

**Comment by leeola** [comment]
Yes, i tried both with antialiasing and without. For what it's worth, the text is very crisp, that is not my problem. My problem is the *thickness* of the letters.

As a comparison, on my screen Monoid's thickness looks similar to Source Code Pro Regular or Source Code Pro Medium. Both of which are very thick, in my opinion. I vastly prefer Source Code Pro Light or Source Code Pro ExtraLight _(hence this request)_.

Not sure how well these will com across on screenshots, but here are some examples:

Source Code Pro Regular

![image](https://cloud.githubusercontent.com/assets/47556/11695439/04b8064e-9e64-11e5-904f-60dbcbaba469.png)

Monoid

![image](https://cloud.githubusercontent.com/assets/47556/11695490/51c678da-9e64-11e5-9ede-a21ddf51cb15.png)

And what i'd like to see, is a thickness similar to this, Source Code Pro ExtraLight _(a Light and ExtraLight would be great, but the below is ExtraLight)_

![image](https://cloud.githubusercontent.com/assets/47556/11695542/9bfe6016-9e64-11e5-9ab1-555683db74f0.png)

The images don't seem to come across that clearly, not sure if they're being compressed by Github or my Screenshot app, but hopefully they help illustrate it, a tiny bit. It's purely preference of course, but that's why i'm asking :)

**Comment by pauldhunt** [comment]
hooray for multiple-weight coding font families! 🙌

**Comment by larsenwork** [comment]
I'll probably start drawing it once I switch to rMBP (waiting for a long overdue update)
I don't really have a use for it now as monoid is all about pixel alignment for coding use. There are many prettier monospaced fonts for text use.

**Comment by astrolemonade** [comment]
@larsenwork can you please consider adding a medium weight to Monoid?
